### PR TITLE
[UI] Improve status indicators

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/connections/helpers.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { CONTROLLER_STATES } from "../../utils/Enum"
+import { LIGHTINDICATORS_CONNECTION_STATES_MAPPING } from "../NotificationCenter/constants"
+
+export const getlightIncicatorForK8s = (operatorState, meshsyncState, brokerState) => { 
+    if (operatorState === CONTROLLER_STATES.DEPLOYED
+        && meshsyncState === CONTROLLER_STATES.CONNECTED
+        && meshsyncState === CONTROLLER_STATES.CONNECTED) {
+            return LIGHTINDICATORS_CONNECTION_STATES_MAPPING.GREEN
```
Comment (MUzairS15) [inline ui/components/connections/helpers.js:8]: Try to map the status with color.

Comment (leecalcote) [reply ui/components/connections/helpers.js:8]: Still draft?

**Comment by leecalcote** [comment]
@MUzairS15 this will soon be 3 weeks old... please tie it off.

**Comment by leecalcote** [comment]
I wonder if we weren't wanting for some of this work during today's huddle.

**Pull request closed without merging**